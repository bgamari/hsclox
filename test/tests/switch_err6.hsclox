switch (5) {
    1+1
}
