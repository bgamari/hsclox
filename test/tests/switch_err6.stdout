[line 2] Error at '1': Expect 'case' before expression.
